src/batou/migrate/migrations/2300.py

```diff
@@ -0,0 +1,17 @@
+def migrate(output):
+    """Manual upgrade steps for 2.3."""
+    output(
+        'Change Address init parameters', """
```

Comment:
i think there's a dedent missing here.

Reply:
No, it is done in `output_migration_step()`